app/src/main/java/com/my/journeyplanner/presenters/main/MainPresenter.kt

```diff
@@ -1,11 +1,11 @@
-package com.my.journeyplanner.presenters
+package com.my.journeyplanner.presenters.main
 
-import com.my.core.domain.JourneyPlannerResult
-import com.my.journeyplanner.framework.Interactors
+import com.my.api.JourneyPlannerResult
+import com.my.api.RetrofitApi
 import com.my.journeyplanner.views.main.MainContract
 import retrofit2.Call
 
-class MainPresenter(private val view: MainContract.View, private val interactors: Interactors) :
+class MainPresenter(private val view: MainContract.View, private val retrofitApi: RetrofitApi) :
```

Comment:
Presenter should not depend on the Api. You need to depend on a Repository, which may be deciding how to deal with the API results (cache if needed etc.)